.circleci/config.yml

```diff
@@ -0,0 +1,130 @@
+version: 2.1
+orbs:
+  ci: bigcommerce/internal@volatile
+  node: bigcommerce/internal-node@volatile
+
+#####################################################
+# Aliases
+#####################################################
+aliases:
+  - &node_executor
+      executor:
+        name: node/node
+        node-version: "10"
+
+  - &release_filter
+      branches:
+        only:
+          - master
+
+  - &pull_request_filter
+      branches:
+        ignore:
+          - master
+
+#####################################################
+# Commands
+#####################################################
+commands:
+  install:
+    steps:
+      - ci/pre-setup
+      - restore_cache:
+          keys:
+            - bigpay-client-js
+      - run: npm ci
+      - save_cache:
+          key: bigpay-client-js
+          paths:
+            - ~/.npm
+  test:
+    steps:
+      - run: npm run prerelease
+  build:
+    steps:
+      - run: npm run prebuild
```

Comment:
NPM should automatically run `prebuild` / `postbuild` when you run `build`. So you shouldn't need to run it separately otherwise it might be executed twice.

Reply:
fixed